tensorflow/lite/kernels/internal/reference/floor_div.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#ifndef TENSORFLOW_LITE_KERNELS_INTERNAL_REFERENCE_FLOOR_DIV_H_
+#define TENSORFLOW_LITE_KERNELS_INTERNAL_REFERENCE_FLOOR_DIV_H_
+
```

Comment:
Can you add #include <cmath> here? The build is failing internally with

```
third_party/tensorflow/lite/kernels/internal/reference/floor_div.h:27:15: error: no member named 'floor' in namespace 'std'
  return std::floor(std::divides<double>()(static_cast<double>(input1),
         ~~~~~^
1 error generated.
```

Reply:
Additional include file now added.